workers/caching/mpcache.toml

```diff
@@ -0,0 +1,8 @@
+main = './mpcache.ts'
+compatibility_date = "2024-09-16"
+
+[env.staging]
+name = "mpcache-staging"
+
+[env.production]
+name = "mpcache"
```

Comment:
Is 2 separate (and differently named) workers the "typical" way to separate QA and production environments?

Reply:
Yup, and can target via the `--env` flag

Reply:
All kinds of various settings can be embedded under here, so its highly configurable per environment.